### ISSUE TITLE
Enclosed alphanumerics and text-align:capitalize

### ISSUE THREAD

**Issue by r12a** [w3c/i18n-activity#222]
**This is a tracker issue.** Only discuss things here if they are i18n WG internal meta-discussions about the issue. **Contribute to the actual discussion at the following link:**

§ https://www.w3.org/Mail/flatten/index?subject=https%3A%2F%2Flists.w3.org%2FArchives%2FPublic%2Fwww-style%2F2015Mar%2F0144.html&list=

**Comment by xfq** [comment]
In https://lists.w3.org/Archives/Public/www-style/2016Oct/0068.html (see the `Capitalizing enclosed alphanumerics (issue #110)` section) the resolution was to leave the spec as-is and not case transform symbols.

**Comment by aphillips** [comment]
(for my [action item](https://www.w3.org/International/track/actions/905)) The result of this thread was the the test was fixed and that lower circled letters do not `text-transform: capitalize` to uppercase circled letters. This transform is similar to Unicode Titlecase, except that it does not include lowercasing of middle/ending letters.

The question that remains is whether this is correct for circled/enclosed letters. There are some weird lacunae in Unicode: the parenthesized letters don't case fold, for example, while the circled letters do. It's a mish-mosh. While my initial personal reaction was that the text-transform should "work as advertised" and provide the case fold for lower case enclosed alphas, in fact the thing that turns out to be arbitrary is Unicode's handling of circled letters.

TL;DR: I think we should close this issue and be satisfied with the result.

**Comment by aphillips** [comment]
Elected to close in https://www.w3.org/2020/05/14-i18n-minutes.html
